ast/ast.go

```diff
@@ -2051,6 +2053,29 @@ type DropTable struct {
 	Name     *Ident
 }
 
+// RenameTable is RENAME TABLE statement node.
+//
+//	RENAME TABLE {{.ToList | sqlJoin ", "}}
+type RenameTable struct {
+	// pos = Drop
+	// end = Name.end
+
+	Rename token.Pos // position of "RENAME" pseudo keyword
+
+	ToList []*RenameTableTo // len(ToList) > 0
```

Comment:
I think `Tos` is not a problem in this case.

Reply:
Thank you.
Fixed. https://github.com/cloudspannerecosystem/memefish/pull/170/commits/527fb7b0a8702b62af4996c469ee3a0f3e60b460